pkg/flow/tracer_map.go

```diff
@@ -22,22 +24,30 @@ type MapTracer struct {
 	evictionTimeout          time.Duration
 	staleEntriesEvictTimeout time.Duration
 	// manages the access to the eviction routines, avoiding two evictions happening at the same time
-	evictionCond   *sync.Cond
-	lastEvictionNs uint64
+	evictionCond               *sync.Cond
+	lastEvictionNs             uint64
+	hmapEvictionCounter        prometheus.Counter
+	numberOfEvictedFlows       prometheus.Gauge
```

Comment:
I'm not sure why this one would be a gauge, this isn't something that will vary up and down, right? Looking at code it's only adding .. so that would rather be a counter